spec/models/adviser_spec.rb

```diff
@@ -0,0 +1,5 @@
+RSpec.describe Adviser, type: :model do
```

Comment:
initial thoughts ...

move `spec/models/adviser_spec.rb` -> `spec/lib/adviser_spec.rb`

- `Advisor` class should respond to csv
- `Office` class should not
- Response of `.to_csv` should return a csv with the headers `'Ref. Number', 'Name', 'Firm'`